rest-api/src/main/resources/definitions/health_data_record.yml

```diff
@@ -27,6 +27,13 @@ properties:
         readOnly: true
         description: JSON map with key value pairs representing the record's data.
         x-nullable: false
+    dayInStudy:
+        type: integer
+        readOnly: true
+        description: |
+            How many calendar days the participant has been in the study. For example, if the participant started on |
+            2019-07-24, then 2019-07-24 is day 1, 2019-07-25 is day 2, etc. Calendar days are calculated using the |
+            same timezone as the Bridge Exporter (ie, America/Los_Angeles).
```

Comment:
x-nullable: false (we will always include it in this record, it will never be missing)?

Reply:
We're not going to backfill this, so it will be missing for old records.

Reply:
Ah, okay.